documentation/modules/known-issues-2-9.adoc

```diff
@@ -0,0 +1,41 @@
+
+.VMware guest virtual machines running Red Hat Enterprise Linux v6.0 cannot boot in {project-short}
+
+VMware guest VMs running Red Hat Enterprise Linux (RHEL) v6.0 cannot boot into the operating system (OS) because the `virtio` disk of RHEL 6.0 requires the `virtio-transitional` device. link:https://issues.redhat.com/browse/MTV-1895[(MTV-1895)]
+
+.Migration plan of virtual machines with NVMe disks hangs in {project-short}
+
+When you select virtual machines (VMs) that use non-volatile memory express (NVMe) disks for creating a migration plan, the plan does not display an error but the migration stalls after the *DiskAllocation* phase because the migration of VMs with NVMe disks is not currently supported. link:https://issues.redhat.com/browse/MTV-2703[(MTV-2703)]
+
+.Source storage is not mapped in the {project-short} user interface for {ocp-short} to {ocp-short} migration
+
+When you select the storage mapping in the plan for a virtual machine for an {ocp-short} to {ocp-short} migration, and update the source storage on the *Storage Maps* page, {project-short} displays a `SourceStorageNotValid` error. link:https://issues.redhat.com/browse/MTV-2784[(MTV-2784)]
+
+.Migration of VMware virtual machine from {ocp-short} to {ocp-short} cluster stalls due to changed name 
+When you migrate a VMware virtual machine (VM) to an {ocp-short} cluster, the VM name is modified. When you try to migrate the VM again from the {ocp-short} cluster to another {ocp-short} cluster, the plan hangs with an `invalid value` error for the VM name. link:https://issues.redhat.com/browse/MTV-2810[(MTV-2810)]
+
+.TPM device of a virtual machine uses a different persistent-state PVC after {ocp-short} to {ocp-short} migration 
+
+When you migrate a virtual machine (VM) with trusted platform module (TPM) from {ocp-short} to {ocp-short}, the VM uses a new persistent volume claim (PVC) that is created for the persistent-state TPM device after migration. link:https://issues.redhat.com/browse/MTV-2838[(MTV-2838)]
+
+.{ocp-short} to {ocp-short} migration of virtual machines with pre-migration or post-migration hooks fail
+
+Cold migration of virtual machines (VMs) from an {ocp-short} cluster to another {ocp-short} cluster fails if the migration plan contains a pre-migration or a post-migration hook. link:https://issues.redhat.com/browse/MTV-2894[(MTV-2894)]
+
+.VDDK image upload failure prevents the creation of VMware provider in the {project-short} user interface 
+When you upload a VMware virtual disk development kit (VDDK) image to the service inventory in the {project-short} user interface, the upload fails with a `Forbidden` error. This prevents you from creating a VMware provider for VMware virtual disk transfers. link:https://issues.redhat.com/browse/MTV-2888[(MTV-2888)]
```

Comment:
I would change the `This prevents you from creating a VMware provider for VMware virtual disk transfers`
to something like:
`This prevents you from creating a VDDK directly from UI, please follow the documentation [link](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.5/html/installing_and_using_the_migration_toolkit_for_virtualization/prerequisites#creating-vddk-image_mtv) to create a VDDK image yourself.`
does not need to be 100% exactly this sentence, but would point them to the docs of workaround.